salt/data-pipeline/bigquery-views.sls

```diff
@@ -0,0 +1,39 @@
+clone biquery-views repo:
+    builder.git_latest:
+        - name: git@github.com:elifesciences/data-pipeline-bigquery-views
+        - identity: {{ pillar.elife.projects_builder.key or '' }}
+        - rev: {{ salt['elife.rev']() }}
```

Comment:
this would mean that this is the primary project deployed on `data-pipeline` nodes, as there can only be one `revision` for each stack. Is this true?
(e.g. other projects would have to point to branches like `master` or find out in some way (there are options) the correct version

Reply:
it's not true, there are other repos there. I'll update this to use the `master` branch

Reply:
done, and the other repos have been switched to `master` as well